plugins/polio/budget/api.py

```diff
@@ -59,6 +59,11 @@ def get_queryset(self) -> QuerySet:
 
     def filter_queryset(self, queryset):
         queryset = super().filter_queryset(queryset)
+
+        org_unit_groups = self.request.query_params.get("orgUnitGroups")
+        if org_unit_groups:
+            queryset = queryset.filter(initial_org_unit__groups__in=org_unit_groups.split(","))
```

Comment:
I think this  should filter on Campaign.country, initial_org_unit is not guaranteed to be a country (the `country` field is automatically calculated from the initial_org_unit)

Reply:
Fixed